html-extract.js

```diff
@@ -41,7 +42,7 @@ module.exports = function (opts) {
           self.push(new gutil.File({
             // Name: id or tag + index.
             path: file.path + "-" + (el.attribs.id || el.tagName + "-" + i),
-            contents: new Buffer(el.children[0].data)
+            contents: new Buffer(trim.left(el.children[0].data))
```

Comment:
Couldn't we just do `(el.children[0].data || "").trim()` and not have an npm dependency on `trim`?